src/libs/Navigation/Navigation.js

```diff
@@ -177,6 +178,19 @@ function getActiveRoute() {
         : '';
 }
 
+/**
+ * @returns {String}
+ */
+function getReportIDFromRoute() {
+    if (!navigationRef.current) {
+        return '';
+    }
+
+    const drawerState = lodashGet(navigationRef.current.getState(), ['routes', 0, 'state']);
+    const reportRoute = lodashGet(drawerState, ['routes', 0]);
+    return lodashGet(reportRoute, ['params', 'reportID'], '');
```

Comment:
NAB: I'm guessing you did this for ultra clarity, but I think it's best to just do a single `lodashGet`.

Reply:
Yep that's the reason. I think it's important for people to think about where these values are coming from. Especially if we have to change up something about the navigators at some point in the future. We could use a code comment to do the same I suppose. I'm finding myself more in the camp of using variables to explain stuff vs having long winded explanations. Not sure if one approach is better than the other or if it's something we should standardize.

Reply:
That's fair. I think I'm coming from mostly recently discovering that `lodashGet` isn't very performant, so I think we should be careful with where we use it. I'm not entirely sure how often this code is triggered to understand if it's a performance concern, but you might know that better than me.

Reply:
I'm not too concerned as this will only be triggered when someone sends you a new comment. That said, I can't guarantee how someone will decide to use it in the future. 

Anyways, feels like the `lodashGet` performance issue would be a good thing to bring up somewhere since the prevailing advice feels like it's becoming: 

>use lodash.get everywhere for consistency... except where it's not performant to do so

The argument for consistency is weakened by this new exception to the rule. I'm a bit concerned that people will start picking up a "lodash bad" mentality without really thinking too hard about why it should or should not be used.

Reply:
Haha, that's fair. Though, I think the opposite is equally bad (ie. lodash get everywhere!) as I've seen it misused way too many times to count. Regardless, I think your use of it here is fine as-is.